tools/certs/root-p256.certspec

```diff
@@ -0,0 +1,8 @@
+issuer:root-p256
```

Comment:
I forgot to mention one thing - when we actually implement this in gecko, I think we don't need to include the root certificate in the signature since it'll be built in to the platform.

Reply:
Makes sense. We ignore the cert chain for now anyway. I plan on doing tests to call this from C++ with a separate API soon-ish. Then this will get interesting.